Apps/WebClient/src/Server/Models/DependentModel.cs

```diff
@@ -35,12 +35,6 @@ public class DependentModel
         [JsonPropertyName("name")]
         public string Name { get; set; } = string.Empty;
 
-        /// <summary>
-        /// Gets or sets the HhdId of the dependent.
-        /// </summary>
-        [JsonPropertyName("hdId")]
-        public string HdId { get; set; } = string.Empty;
```

Comment:
One this PR gets merged to DEV, we will see if we will have broken build as I think it will delete twice.